tests/test_minhash.py

```diff
@@ -1520,6 +1520,42 @@ def test_flatten():
     assert len(mh2) == 3
 
 
+def test_inflate():
+    # test behavior with scaled
+    scaled = _get_scaled_for_max_hash(35)
+    mh = MinHash(0, 4, track_abundance=True, scaled=scaled)
+    mh2 = MinHash(0, 4, track_abundance=True, scaled=scaled)
+    assert mh._max_hash == 35
+
+    mh.add_hash(10)
+    mh.add_hash(20)
+    mh.add_hash(30)
+
+    assert mh.hashes[10] == 1
+    assert mh.hashes[20] == 1
+    assert mh.hashes[30] == 1
+
+    mh2.add_hash(10)
+    mh2.add_hash(10)
+    mh2.add_hash(10)
+    mh2.add_hash(20)
+    mh2.add_hash(20)
+    mh2.add_hash(30)
+    mh2.add_hash(30)
+    mh2.add_hash(30)
+
+    assert mh2.hashes[10] == 3
+    assert mh2.hashes[20] == 2
+    assert mh2.hashes[30] == 3
+
+    mh.inflate(mh2)
```

Comment:
this should be something like `mh3 = mh.inflate(mh2)`. Right now `inflate` is modifying `mh` in place, but per the original issue, "this method should create a new MinHash object" - and, while I did not say so explicitly, it should return that object :).

Reply:
Oh okay got it :)